desktop/php/jMQTT_broker.php

```diff
@@ -112,6 +112,27 @@
                                     autocomplete="off" style="margin-top: 5px" placeholder="jeedom" />
                             </div>
                         </div>
+
+                        <div class="form-group">
+                            <label class="col-lg-4 control-label">{{Activer TLS : }}</label>
```

Comment:
It would be better to have a ComboBox (l2key=mqttTlsMode ?) with 3 possibilities : 
0=NoTls (Désactivé)
1=TlsUsingSystemCAs (Activé - Authorités Publiques)
2=TlsUsingCustomCA (Activé - Authorités Personnalisées)